lib/laa_crime_schemas/structs/investment.rb

```diff
@@ -4,8 +4,8 @@ module LaaCrimeSchemas
   module Structs
     class Investment < Base
       attribute :investment_type, Types::InvestmentType
```

Comment:
Is this meant to be optional?

Reply:
This can xxx_type can never not exist as there is a database constraint on Apply